sopel_waifu/waifu.json

```diff
@@ -1453,37 +1506,112 @@
         "Yuika Mitsumine"
     ],
     "The Tatami Galaxy": [
-        "Akashi"
+        "Akashi",
+        "Hanuki Ryouko",
+        "Higuchi Keiko",
+        "Kaori"
     ],
     "To Aru Kagaku no Railgun": [
-        "Shirai Kuroko",
         "Saten Ruiko",
-        "Uiharu Kazari"
+        "Kiyama Harumi",
+        "Konori Mii",
+        "Juufuku Miho",
+        "Tessou Tsuduri",
+        "Wannai Kinuho",
+        "Awatsuki Maaya",
+        "Haruue Erii",
+        "Therestina Kihara Lifeline",
+        "Edasaki Banri",
+        "Kounoe",
+        "Hazamaya Kana",
+        "Nunotaba Shinobu",
+        "Frenda Seivelun",
+        "Mugino Shizuri",
+        "Takitsubo Rikou",
+        "Shokuhou Misaki",
+        "Kinuhata Saiai",
+        "Minori",
+        "Febri",
+        "Hokaze Junko",
+        "MISAKA 1"
+        "Kumokawa Seria",
+        "Xochitl",
+        "Taroumaru Seike",
+        "Yakumaru Itsuki",
+        "Iizumi Rita",
+        "Sakuragi Naruha",
+        "Dolly",
+        "Kouzaku Mitori",
+        "Kuriba Ryouko",
+        "Yumiya Rakko"
     ],
     "To Aru Majutsu no Index": [
-        "Index",
-        "Last Order",
+        "Index Librorum Prohibitorum",
         "Misaka \"Biribiri\" Mikoto",
-        "Misaka Imouto",
+        "Kanzaki Kaori",
         "Tsukuyomi Komoe",
+        "Kuroko Shirai",
         "Himegami Aisa",
+        "MISAKA 10032",
+        "Tsuchimikado Maika",
+        "Sasha Kreuzhev",
+        "Last Order",
+        "Kazakiri Hyouka",
+        "Itsuwa",
+        "Uiharu Kazari",
+        "Sherry Cromwell",
+        "Yomikawa Aiho",
+        "Yoshikawa Kikyou",
+        "Ryoukan",
+        "Laura Stewart",
+        "Kamijou Shiina",
+        "MISAKA 10031",
+        "MISAKA 19090",
         "Fukiyose Seiri",
-        "Mitsuko Kongo"
+        "Mitsuko Kongo",
+        "Vento of the Front",
+        "Oriana Thomson",
+        "Lidvia Lorenzetti",
+        "Lucia",
+        "Angelene",
+        "Agnese Sanctis",
+        "Orsola Aquinas",
+        "Misaka Misuzu",
+        "Musujime Awaki",
+        "Vasilisa",
+        "Uragami"
     ],
     "Tonari no Kaibutsu-kun": [
-        "Shizuku \"Mitti\" Mizutani",
-        "Asako Natsume"
+        "Mizutani \"Mitti\" Shizuku",
+        "Natsume Asako",
+        "Ooshima Chizuru",
+        "Miyama Yuu",
+        "Ninomiya Saeko",
+        "Yumeko"
     ],
     "Top wo Nerae!": [
-        "Takaya Noriko"
+        "Takaya Noriko",
+        "Amano Kazumi",
+        "Jung Freud",
+        "Kashiwara Reiko"
     ],
     "Top wo Nerae! 2": [
-        "Nono"
+        "Nono",
+        "Lal'C Melk Mark",
+        "Tycho Science"
     ],
```

Comment:
Thoughts on combining these into a single "Top wo Nerae!" entry? I don't see a lot of point in keeping "Show" listed separately from a sequel "Show 2".

Reply:
Well then you'd probably have to merge all the IM@S by that logic wouldn't you? And Index and it's various spinoffs etc.

Reply:
iM@S specifically is safe AFAIK because each game/show is self-contained. There's little to no crossover/carryover of characters from one to the next. Same applies to e.g. _Love Live!_'s various generations. I say we should draw a line between such things and sequels that carry mostly the same cast through (like "Top wo Nerae!", "Hibike! Euphonium", etc.)

The _Index_ universe is complicated, and that's why it has its own line note below. It has both sequels _and_ spin-offs, with supporting/minor characters from the original series becoming _main_ characters in the derived works.

Reply:
Right, well that applies to "Top Wo Nerae" as well since it's self-contained too. IIRC they take place some decades apart.

Reply:
Then that's settled. These stay separate.